app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  # GreetingsController
+  class GreetingsController < ApplicationController
+    def index
+      greetings = Greeting.find(Random.new.rand(1..5))
+      render json: greetings
+    end
+  end
+end
```

Comment:
- [ ] Kindly fix that issue, this makes your project not display random messages/greetings.

![image](https://user-images.githubusercontent.com/55635977/166338912-9c06a797-3627-41e9-9941-a67b36531ddc.png)

Reply:
@Emmyn5600 I this that because you didn't add a message with that id, The requirements say `Create a table for storing your messages and create 5 different greetings.` and display one of them, that is what I did choose one random from 1 to 5.

Reply:
@Emmyn5600 You can run `rails db:seed` to get the initial values.

Reply:
@keroloussamy, to avoid all these misunderstanding, we can document every commands or instructions in the readme file, `rails db:migrate rails db:seed`